xtremedata-dbx-cluster-centos/azuredeploy.json

```diff
@@ -0,0 +1,452 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "String",
+            "defaultValue": "West US",
+            "allowedValues": [
+                "West US"
+            ],
+            "metadata": {
+                "description": "Azure cloud location for dbX cluster"
+            }
+        },
+        "newStorageAccountName": {
+            "type": "String",
+            "defaultValue": "dbxstrg",
```

Comment:
Please remove default value for storage account name as it needs to be globally unique